src/Symfony/Component/Serializer/Mapping/Loader/YamlFileLoader.php

```diff
@@ -103,6 +103,26 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                     $attributeMetadata->setIgnore($data['ignore']);
                 }
 
+                foreach ($data['mappings'] ?? [] as $line) {
+                    $groups = $line['groups'] ?? [];
+
+                    if ($serializedName = $line['serialized_name'] ?? false) {
+                        if (!\is_string($serializedName) || '' === $serializedName) {
```

Comment:
What about `' '`?

Reply:
Could you please elaborate? Do I need to trim string and then check for emptiness? Or just check for single space? What's the point?